ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureV2ButtonCustomization.java

```diff
@@ -62,15 +111,35 @@ public int describeContents() {
 
     @Override
     public void writeToParcel(Parcel parcel, int i) {
-        super.writeToParcel(parcel, i);
-        parcel.writeString(backgroundColor);
-        parcel.writeInt(cornerRadius);
+        parcel.writeString(cardinalButtonCustomization.getTextFontName());
+        parcel.writeString(cardinalButtonCustomization.getTextColor());
+        parcel.writeInt(cardinalButtonCustomization.getTextFontSize());
+        parcel.writeString(cardinalButtonCustomization.getBackgroundColor());
+        parcel.writeInt(cardinalButtonCustomization.getCornerRadius());
     }
 
     protected ThreeDSecureV2ButtonCustomization(Parcel in) {
-        super(in);
+        textFontName = in.readString();
+        textColor = in.readString();
+        textFontSize = in.readInt();
         backgroundColor = in.readString();
         cornerRadius = in.readInt();
+
+        if (textFontName != null) {
```

Comment:
Our parcelable implementation was setting member variables in these classes, but not setting the customizations on the actual cardinal customization class. Updated the implementations to set the cardinal customizations too.